doc/api.md

```diff
@@ -537,6 +538,33 @@ which is `details.internal_name` and falls back to `title`.
 - `document_type` *(required)*
   - The `document_type` value that returned editions has.
 
+## `GET /v2/editions`
+
+Retrieves a paginated list of editions for the provided query string
+parameters.
+
+### Query string parameters
+
+- `fields[]` *(optional)*
+  - Accepts an array of: "analytics_identifier", "base_path",
+    "content_id", "description", "document_type", "locale",
+    "public_updated_at", "schema_name", "title"
+  - Determines which fields will be returned in the response, if omitted all
+    fields will be returned.
+- `locale` *(optional)*
+  - Accepts: An available locale from the [Rails I18n gem][i18n-gem]
+  - Used to restrict documents to a given locale.
+- `key` *(optional, defaults to `updated_at,id`)
```

Comment:
Can we change this to order and use it in the same way as [get content](https://github.com/alphagov/publishing-api/blob/master/doc/api.md#user-content-query-string-parameters)

I think the id can be implicit and not something that needs be part of a users concern.